kolibri/plugins/management/assets/src/vue/class-enroll-page/index.vue

```diff
@@ -1,30 +1,357 @@
 <template>
 
   <div>
-    <p>enroll for class {{ this.classId}}</p>
+    <div class="top-buttons pure-g">
+
+      <div :class="windowSize.breakpoint > 2 ? 'pure-u-1-2' : 'pure-u-1-1 align-center'">
+        <router-link :to="editClassLink">
+          <icon-button
+            :text="$tr('backToClassDetails')"
+            :primary="false">
+            <mat-svg category="navigation" name="arrow_back"/>
+          </icon-button>
+        </router-link>
+      </div>
+
+      <div :class="windowSize.breakpoint > 2 ? 'pure-u-1-2 align-right' : 'pure-u-1-1 align-center'">
+        <icon-button
+          :text="$tr('enrollSelectedUsers')"
+          :primary="true"
+          @click="openConfirmEnrollmentModal"
+          :disabled="selectedUsers.length === 0">
+          <mat-svg category="navigation" name="check"/>
+        </icon-button>
```

Comment:
First issue, is an issue with the modal focus that exists not just in this page @DXCanas 
And you were not testing the latest @indirectlylit . That was the `ui-confirm` which I replaced with the `core-modal`.